test/src/596-catalogbalancer/main

```diff
@@ -0,0 +1,156 @@
+cvmfs_test_name="Catalog balancer"
+cvmfs_test_autofs_on_startup=false
+
+add_files() {
+  local repo_dir=$1
```

Comment:
It's a bit confusing to call this `repo_dir`, too.  It's a directory within the repository, no?